docs/the-new-architecture/use-app-template.md

```diff
@@ -1,7 +1,134 @@
 ---
 id: use-app-template
-title: Using the New App Template
+title: Creating a New Architecture App
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+import RemoveGlobalCLI from '.././\_remove-global-cli.md';
+
+This page will help you create a new React Native app that uses the New Architecture.
+
+## Development environment
+
+Before continuing, make sure you've followed all the steps in [Setting up the
+development environment](getting-started.md), under the **React Native CLI
+Quickstart** tab.
+
+If following the setup guide, stop when you reach the section **Running your React Native
+Application**, and resume following this guide.
```

Comment:
```suggestion
If following the setup guide, stop when you reach the section **Running your React Native Application**, and resume following this guide.
```

Reply:
> A question: are you using some tools to write the guide? I found several sentences with a new line in the middle of the sentence which could lead to bugs (missing whitespaces mainly). It really looks like a sort of word wrap feature. 

Sort of, I manually wrapped paragraphs that were getting very long -- it was just a habit from writing Markdown docs for other purposes, I'll fix it 😬 thanks for catching!